libqtile/widget/prompt.py

```diff
@@ -25,7 +25,7 @@
 #
 # THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 # IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVhENT SHALL THE
```

Comment:
I'm very sorry, i believe it has to be very annoying for you to correct those things after me.
I'll try be more careful next-time. Fixed. Thank you.